src/transformers/models/dpr/modeling_tf_dpr.py

```diff
@@ -299,6 +300,97 @@ def call(
         )
 
 
+class TFDPRSpanPredictor(TFPreTrainedModel):
+
+    base_model_prefix = "encoder"
+
+    def __init__(self, config: DPRConfig, **kwargs):
+        super().__init__(config, **kwargs)
+        self.encoder = TFDPRSpanPredictorLayer(config)
+
+    def call(
+        self,
+        input_ids: tf.Tensor = None,
+        attention_mask: Optional[tf.Tensor] = None,
+        token_type_ids: Optional[tf.Tensor] = None,
+        inputs_embeds: Optional[tf.Tensor] = None,
+        output_attentions: bool = False,
```

Comment:
(note) in PT function args like `output_attentions` default to `None`, in TF a lot of inputs still default to False/True...maybe we should think about aligning this at some point

Reply:
Good point! I add this to my todo list.